package.js

```diff
@@ -3,13 +3,20 @@ Package.describe({
   name: "practicalmeteor:mocha",
   summary: "Write package tests with mocha and run them in the browser or from the command line with spacejam.",
   git: "https://github.com/practicalmeteor/meteor-mocha.git",
-  version: '2.1.0_7'
+  version: '2.1.1-rc.1',
+  testOnly: true
+});
+
+
+Npm.depends({
```

Comment:
It's not required but it seems like a better approach, given the tool is basically doing it's own "browserifying".